core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -220,8 +220,8 @@ class SearchService(
     backlogActor ! FileUpdate(f, symbols)
   }(workerEC)
 
-  def shutdown(): Unit = {
-    db.shutdown()
+  def shutdown()(implicit ec: ExecutionContext): Unit = {
+    db.shutdown
```

Comment:
oh, looks like someone changed it to only use one execution context. Yeah, no need for explicit passing.